charts/nautobot/templates/nautobot-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- $config_checksum := dict "checksum/config" (print $.Template.BasePath "/configmap.yaml" . | sha256sum) "checksum/secret" (print $.Template.BasePath "/secret.yaml" . | sha256sum) }}
```

Comment:
I tested it locally with: `helm template . -s templates/nautobot-deployment.yaml`

It calculates different checksums every time we run the command. I am not sure if this is something we want.

Reply:
We can probably add something like this:

```
{{- $config_checksum := (include "nautobot.configMap.env" . | sha256sum | quote) }}

  template:
    metadata:
      annotations:
        test-checksum: {{ $config_checksum }}
```

Reply:
```
#
# values.yaml: allowedHosts: "*"
#
$ helm template . | grep "test-checksum"
        test-checksum: "a5d5bf5cc1e3c3fddd81f11114372e9559d8f9637309c8f28b4b5bea817f2068"
#
# values.yaml: allowedHosts: "foobar"
#
$ helm template . | grep "test-checksum"
        test-checksum: "b6b470bf60067da5c63d75bd0e3bca0f2a58decea40076721934ec60fe1b5288"
#
# values.yaml: allowedHosts: "*"
#
$ helm template . | grep "test-checksum"
        test-checksum: "a5d5bf5cc1e3c3fddd81f11114372e9559d8f9637309c8f28b4b5bea817f2068"
```

Reply:
I see your point. 
So I used the include method with the values and not checking the files, cause every time the files are rendered a new sha256 hash was generated. 
I also removed secrets totally for now.

Reply:
Yes I think that makes a lot of sense.

Reply:
Actually tried it and it doesn't work so good because we use `common.names.fullname` into the config map and those are coming with nil values on rendering evaluation.